Clinical-Genomics/cgweb
# Fixes problem submitting external samples

This PR fixes the problem editing source on external samples

How to test:
0. install on stage on clinical-api cgweb
1. login
1. create external samples order
1. add sample
1. fill in all data
1. press save 

Expected outcome:
Sample edit view is minimized to a row for the sample
